markets/storageadapter/provider.go

```diff
@@ -229,6 +230,7 @@ func (n *ProviderNodeAdapter) LocatePieceForDealWithinSector(ctx context.Context
 	if bestSi.State == sealing.UndefinedSectorState {
 		return 0, 0, 0, xerrors.New("no sealed sector found")
 	}
+	// TODO(review) is this offset actually padded?
```

Comment:
Yes, did go through a bunch of levels, and it seems to be used correctly everywhere

```suggestion
```